### ISSUE TITLE
xssf load error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sshyran/xssf#17]
```
Can someone help me out, I have got error:

Failed to load plugin from /opt/metasploit/apps/pro/msf3/plugins/xssf: cannot 
load such file -- /opt/metasploit/apps/pro/msf3/plugins/xssf

I installed xssf with MSF 4.6.0-dev and Backtrack 5R3.

Thanks a lot!
```

Original issue reported on code.google.com by `victordu...@gmail.com` on 15 Apr 2013 at 9:24

**Comment by GoogleCodeExporter** [comment]
```
Hello,
That not seem to be a problem related to XSSF source code, but more a problem 
related to XSSF installation.

Are you sure you copied all files in the right drectory ? You should copy all 
files within /opt/metasploit/apps/pro/msf3/ directory.

Is 'xssf.rb' file included within the /opt/metasploit/apps/pro/msf3/plugins/ 
directory ?
```

Original comment by `ludovic....@gmail.com` on 23 Apr 2013 at 5:46

**Comment by GoogleCodeExporter** [comment]
```
Thanks for you reply, I have solved it out using kali linux 1.0.
```

Original comment by `victordu...@gmail.com` on 23 Apr 2013 at 7:40

**Comment by GoogleCodeExporter** [comment]
Original comment by `ludovic....@gmail.com` on 22 May 2013 at 10:49
- Changed state: **Fixed**
